index.js

```diff
@@ -1,10 +1,65 @@
+// HTML
 const formElement = document.getElementById("formMatchs");
+const matchListElement = document.getElementById("matchsList");
+
+
+// Array 
+const matchList = [];
+
 
 function getDataFromForm(event) {
     event.preventDefault();
-    new FormData(formElement).forEach(console.log)
+    const dataForm = new FormData(formElement);
+    const teamA = dataForm.get("teamA");
+    const teamB = dataForm.get("teamB");
+    const scoreTeamA = dataForm.get("scoreTeamA");
+    const scoreTeamB = dataForm.get("scoreTeamB");
+
+    addMatchToMatchList({ teamA, teamB, scoreTeamA, scoreTeamB });
+    formElement.reset();
 }
 
+function isMatchValid({ teamA, teamB, scoreTeamA, scoreTeamB }) {
+    if (teamA.trim() === "" || teamB.trim() === "") {
+        return false;
+    }
+    if (teamA === teamB) {
+        return false;
+    }
 
-formElement.onsubmit = getDataFromForm;
+    return true;
+
+    // Team A != Team B ; => surtout ca 
+}
+
+function addMatchToMatchList({ teamA, teamB, scoreTeamA, scoreTeamB }) {
+    let winner;
+
+    if (scoreTeamA > scoreTeamB) {
+        winner = teamA;
+    }
 
+    if (scoreTeamB > scoreTeamA) {
+        winner = teamB;
+    }
+
+    if (isMatchValid({ teamA, teamB, scoreTeamA, scoreTeamB })) {
+        matchList.push({ teamA: teamA, teamB: teamB, winner: winner });
+        updateMatchListElement();
+    } else {
+        alert("Match isn't valid");
+    }
+}
+
+function updateMatchListElement() {
+    const lastElementFromArray = matchList[matchList.length - 1];
+
+    const newMatchItem = document.createElement("li");
+    const itemText = document.createTextNode(lastElementFromArray.teamA + " VS " + lastElementFromArray.teamB + " Winner is => " + lastElementFromArray.winner);
```

Comment:
et si ya pas de winner 😄  -> ca fait un truc du genre `Winner is -> null`

je pense qu'on arrive a mieux faire :-)

Reply:
Justement c'est se que j'ai suggérer à jérôme mais il m'a dit que y'a pas forcément besoin

Reply:
A moins que j'ai mal écouté